adserver/templates/adserver/base.html

```diff
@@ -42,6 +42,10 @@
               <span class="fa fa-key fa-fw mr-2 text-muted" aria-hidden="true"></span>
               <span>{% trans 'Change password' %}</span>
             </a>
+            <a class="dropdown-item" href="{% url 'mfa_index' %}">
+              <span class="fa fa-unlock-alt fa-fw mr-2 text-muted" aria-hidden="true"></span>
+              <span>{% trans 'MFA' %}</span>
```

Comment:
This should probably be spelled out? 

```suggestion
              <span>{% trans 'Multi-factor authentication (MFA)' %}</span>
```

Reply:
This doesn't fit here but I spelled out "Two-factor Auth" and then defined the acronym on the 2FA detail screen.